README.md

```diff
@@ -41,15 +41,15 @@ This is a SIMPLE bashdb debugger frontend. Useful for learning bash shell usage
 Hopefully bash will land on 3 leading platforms making this extension very useful. Useful hint: shellcheck extension does a great job with finding common script errors before debugging.
 
 ## Usage
-1. Select "Debug -> Start Debugging (F5)" to start debugging (if launch.json is not available, it will be created with default configuration)
-2. Select "Debug -> Add Configuration" to add custom debug configuration (drop-down, path-input, etc...)
+1. Select `Debug -> Start Debugging (F5)` to start debugging (if launch.json is not available, it will be created with default configuration)
```

Comment:
`Debug -> Start Debugging (F5)` is certainly readable, but it may be _italic_ or **bold**, because it doesn't appear in any `monospaced` context (on command prompt, in text editor, or so on).
This guid is not stipulated rule, but I think it's convention as far as I know.